custom_visibility/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -194,7 +232,15 @@ public function elasticProcessDocumentPreIndex(Document $document, \SugarBean $b
     }
 
     /**
-     * {@inheritdoc}
+     * implementation of ElasticStrategyInterface interface
+     *
+     * Bean index fields to be indexed
```

Comment:
I think this would be more clear if it were a verb phrase.  As written, it sounds like a param description.